Makefile

```diff
@@ -27,6 +27,7 @@ EMACS := $(shell which "$${EMACS}" || which "emacs")
 EMACS_VERSION := $(shell "$(EMACS)" -Q --batch --eval '(princ emacs-version)')
 
 EFLAGS = --eval "(add-to-list 'load-path (expand-file-name \"tests/compat\") 'append)" \
+     --eval "(load-file (expand-file-name \"tests/haskell-test-utils.el\")" \
```

Comment:
This line is not needed. You have `require` in individual files that will source `haskell-test-utils.el`.